public/docs/ts/latest/guide/template-syntax.jade

```diff
@@ -542,7 +542,7 @@ table
 // #docregion property-binding-7
 :marked
   ### Binding target
-  a id between enclosing square brackets identifies the target property. The target property in the following code is the image element’s `src` property.
+  An element property between enclosing square brackets identifies the target property. The target property in the following code is the image element’s `src` property.
```

Comment:
I see this, but I would say `An attribute between enclosing...`.

But that is my opinion. Let see what @wardbell has to say.